### PR TITLE
Add details about how to add a new dashboard in Kibana

### PR DESCRIPTION
**Pull request by monicasarbu**

### CODE REVIEW

**Comment by monicasarbu** [comment]
@dedemorton I added a section in the readme file about how to create your own dashboard. Can you please have a look?

**Comment by ruflin** [comment]
LGTM.

File: README.md [hunk 1]
```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
...
+load.sh script. Then, create the Kibana dashboard using Kibana functionalities
+and then download all the dashboards using the save/kibana_dump.py script. 
+
+Note: Make sure you are using the latest Kibana version to create and download
+the dashboards.
+
+Before executing the save/kibana_dump.py script, make sure you have python and virtualenv
+installed:
+
+        # Prepare the envirement
```
Comment (andrewkroh) [inline README.md]: I recommend to explicitly say `cd save` in these steps.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
 shell script so that you don't need python installed when loading the
 dashboards.
 
+
+Create a new dashboard
+----------------------
+
+If you added support for a new protocol in Packetbeat or a module in
+Metricbeat, it would be nice to create a dedicated Kibana dashboard to 
+prove the functionality of your data.
```
Comment (dedemorton) [inline README.md]: Suggest changing "prove the functionality of your data" to "visualize your data" (I'm not really sure what "prove the functionality of your data" means).

File: README.md [hunk 3]
```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
...
+The Kibana dashboards are saved in a special index in Elasticsearch. By default
+it's .kibana, but it can be configured to anything else.
+
+The first step, in creating your own Kibana dashboard, would be to load
```
Comment (dedemorton) [inline README.md]: The first step in creating your own Kibana dashboard is to load all the current dashboards into Kibana by using the
`load.sh` script. [you might also want to add a link to the doc about running the script?]

File: README.md (hunk 1 above)
Comment (dedemorton) [inline README.md]: typo: envirement > environment

File: README.md [hunk 4]
```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
...
+
+If you added support for a new protocol in Packetbeat or a module in
+Metricbeat, it would be nice to create a dedicated Kibana dashboard to 
+prove the functionality of your data.
+The Kibana dashboards are saved in a special index in Elasticsearch. By default
+it's .kibana, but it can be configured to anything else.
+
+The first step, in creating your own Kibana dashboard, would be to load 
+all the current dashboards in Kibana using the
+load.sh script. Then, create the Kibana dashboard using Kibana functionalities
```
Comment (dedemorton) [inline README.md]: This part is bit hard to follow. The phrase "using Kibana functionalities" seems a little vague, and users probably aren't familiar with the kibana_dump.py script (I'm not). I understand that I'm supposed to use Kibana to create a new dashboard, but I need a few more details. For example, why do I need to load the other dashboards first? Is it mainly so that the index patterns get created? Am I expected to somehow build visualizations for my new dashboard by looking at the sample dashboards?

File: README.md [hunk 5]
```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
...
+and then download all the dashboards using the save/kibana_dump.py script. 
+
+Note: Make sure you are using the latest Kibana version to create and download
```
Comment (dedemorton) [inline README.md:52]: Seems like this note should come earlier.

File: README.md [hunk 6]
```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
...
+
+        # Prepare the envirement
+        virtualenv env
+        . env/bin/activate
+        pip install -r requirements.txt
+
+        # Download all Kibana dashboards to your host
+        python kibana_dump.py --url 'http://localhost:9200' --dir output
+
+where url points to the Elasticsearch URL, dir is the directory where to place
```
Comment (dedemorton) [inline README.md]: where `url` points to the Elasticsearch URL, and `dir` is the directory where you want to save the Kibana dashboards.

File: README.md [hunk 7]
```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
...
+the Kibana dashboards.
+
+As last steps, copy the related dashboards, visualizations, searches and
+eventually index patterns to the dashboards directory and send us a pull request
```
Comment (dedemorton) [inline README.md]: Finally, copy the related dashboards, visualizations, searches, and eventually index patterns to the `dashboards` directory, and send us a pull request. [I'd put directory names in backticks--or whatever formatting works on GitHub--so that they are distinguishable from regular text.]

Comment (dedemorton) [inline README.md]: @monicasarbu I'm done adding comments.

Comment (monicasarbu) [inline README.md]: Thanks a lot, @dedemorton

**Comment by monicasarbu** [comment]
I am done with my changes.
